libfabric.map.cpp.in

```diff
@@ -13,5 +13,9 @@ FABRIC_1.0 {
 		fi_param_get;
 		fi_getparams;
 		fi_freeparams;
+#ifdef FI_DIRECT_ENABLED
+#include "prov/@PROVIDER_DIRECT@/provider.map"
+#endif
 	local: *;
 };
```

Comment:
Also, please check out this when configuring/building outside of source tree. https://github.com/ofiwg/libfabric/commit/6baf1f3b8b71f104aec477a9c9a5edf09017a9b2